src/oc/web/components/comments.cljs

```diff
@@ -25,7 +47,18 @@
           [:div.comment-author
             (:name author)]
           [:div.comment-timestamp
-            (utils/time-since (:created-at c))]]]
+            (utils/time-since (:created-at c))]]
+          (when (seq (:links c))
```

Comment:
@thepug  i think you should check if `(utils/link-for (:links c) "delete")` is present instead of checking if there are links. This may be a problem if more links are added.